### PR TITLE
Updated governance, contributing, and code of conduct

### PR DESCRIPTION
**Pull request by CoralineAda**
## Summary

1. Changes to the governance model and code of conduct related to the change in stewardship of Contributor Covenant.
2. Updated and reorganized the contributing guide.

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -1,37 +1,38 @@
...
 
 ---
 
-[![CommunityRule derived](https://communityrule.info/assets/CommunityRule-derived-000000.svg)](http://communityrule.info)
+Created by [Organization for Ethical Source](https://ethicalsource.dev)
 
-Created with [CommunityRule](http://communityrule.info)   
-[Creative Commons BY-SA](http://creativecommons.org/licenses/by-sa/4.0/).
+[![CommunityRule derived](https://communityrule.info/assets/CommunityRule-derived-000000.svg)](https://communityrule.info)
+[Creative Commons BY-SA](https://creativecommons.org/licenses/by-sa/4.0/)
```
Comment (berax574) [inline GOVERNANCE.md:38]: Yxyxucuctsyfuci

**Review by berax574 — CHANGES_REQUESTED** [review]

**Comment by tuxtvixx** [comment]
Nice..

**Comment by tuxtvixx** [comment]
> ## Summary
> 
> 1. Changes to the governance model and code of conduct related to the change in stewardship of Contributor Covenant.
> 2. Updated and reorganized the contributing guide.
>

**Comment by tuxtvixx** [comment]
Nice..

**Comment by tuxtvixx** [comment]
Fantastic.. One more step behind..

**Pull request merged**